tests/integration-tests/benchmarks/test_scaling_performance/test_scaling_performance/pcluster.config.ini

```diff
@@ -0,0 +1,25 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = 0
+max_queue_size = 500
```

Comment:
Let's say we wanted to do > 500 nodes, can we set this to `scaling_target` instead of 500?

Reply:
yes but let's put it to scaling_target + 10 in order to detect overscaling